build/DependencyVersions.props

```diff
@@ -17,6 +17,8 @@
     <PlatformAbstractionsVersion>2.0.0-preview2-002093</PlatformAbstractionsVersion>
     <DependencyModelVersion>2.0.0-preview2-002093</DependencyModelVersion>
     <CliCommandLineParserVersion>0.1.0-alpha-142</CliCommandLineParserVersion>
+    <CliProjectJsonMigration>1.1.0-alpha-2127</CliProjectJsonMigration>

```

Comment:
Which two? CommandLine and ProjectJsonMigration? Why would they be the same? Don't we have a commit count in there? Looks like neither of these are correctly zero padding. I think that if both are going to ship with CLI 2.0.0, then everything up to the buildnumber/commit count could be the same as CLI 2.0.0, just as with dotnet/sdk.